src/js/protocols/stm32usbdfu.js

```diff
@@ -333,10 +333,20 @@ STM32DFU_protocol.prototype.getChipInfo = function (_interface, callback) {
             // F72x: "@Internal Flash  /0x08000000/04*016Kg,01*64Kg,03*128Kg"
             // F74x: "@Internal Flash  /0x08000000/04*032Kg,01*128Kg,03*256Kg"
 
-            // H750 SPRacing H7 EXST: "@External Flash /0x90000000/998*128Kg,1*128Kg,4*128Kg,21*128Ka"
-            // H750 SPRacing H7 EXST: "@External Flash /0x90000000/1001*128Kg,3*128Kg,20*128Ka" - Early BL firmware with incorrect string, treat as above.
+            // H750 SPRacing H7 EXST
+            // Descriptors:
+            // "@External Flash /0x90000000/998*128Kg,1*128Kg,4*128Kg,21*128Ka" (standard)
+            // "@External Flash /0x90000000/1001*128Kg,3*128Kg,20*128Ka" (Early BL firmware with incorrect string, treat as above)
+            // Partitions:
+            // Flash, Config, Firmware, 1x BB Management block + x BB Replacement blocks
+
+            // H730 SPRacing H7 EXST with SPRacingPixelOSD
+            // Descriptors:
+            // "@External Flash /0x90000000/111*8Kg,16*8Kg,1*8Kg,128*8Kg,0*8Ka"
+            // Partitions:
+            // OSD data (fonts, logos, etc), System Partition, Config, Firmware, BB Management block + x BB Replacement blocks
+
 
-            // H750 Partitions: Flash, Config, Firmware, 1x BB Management block + x BB Replacement blocks)
             if (str == "@External Flash /0x90000000/1001*128Kg,3*128Kg,20*128Ka") {
                 str = "@External Flash /0x90000000/998*128Kg,1*128Kg,4*128Kg,21*128Ka";
             }
```

Comment:
What is this? A bug in the FC? I don't understand how this works (the flash system). I suppose this cannot be fixed in any way and this is a workaround in the Configurator to support it?

Reply:
I close this because the only change was the commit, so we are already supporting this.